src/plugins/zotero/editor/ZoteroEditor.jsx

```diff
@@ -6,6 +6,7 @@ import { setPluginOptions } from '@plone/volto-slate/actions';
 import { ZoteroEditorSchema } from './schema';
 import ZoteroDataWrapper from './ZoteroDataWrapper';
 
+// eslint-disable-next-line import/no-anonymous-default-export
```

Comment:
@ana-oprea We should avoid as much as possible `eslint-disable` comments to fix eslint issues

Reply:
@avoinea I commited the fix